notifications/workflow-email/classes/TestRunEmailController.cls

```diff
@@ -7,50 +7,43 @@ public class TestRunEmailController {
     }
 
     public Map<String, List<Test_Run_Method_Result__c>> getFirstFailureMap() {
-        Map<String, List<Test_Run_Method_Result__c>> firstFailureMap = getMethodResults(true);
-        return firstFailureMap;
-    }
-    public Map<String, List<Test_Run_Method_Result__c>> getRemainingFailureMap() {
-        Map<String, List<Test_Run_Method_Result__c>> remainingFailureMap = getMethodResults(false);
-        return remainingFailureMap;
-    }
-
-    private Map<String, List<Test_Run_Method_Result__c>> getMethodResults(Boolean firstFailure) {
-        queryResults = [
-            SELECT Id, Name, Method_Name__c, Message__c, First_Failure__c
-            FROM Test_Run_Method_Result__c
-            WHERE Test_Run__c = :testRunId AND Method_Pass__c = false
-        ];
+        if (queryResults == null) {
```

Comment:
@fishygeek91 
I forgot that VF components are weird and the attributes aren't set until AFTER the constructor is called.

This is fine, but I think using GETTERs is a bit cleaner. 

https://salesforce.stackexchange.com/questions/9941/setter-method-in-vf-component-being-called-after-the-constructor-has-retured